packages/web/src/utils/web.config.spec.ts

```diff
@@ -74,24 +73,6 @@ describe('getWebConfig', () => {
     ).toEqual('style-loader');
   });
 
-  describe('statsJson', () => {
```

Comment:
The `StatsPlugin` was removed here https://github.com/angular/angular-cli/commit/06c1e1c05152f199e65dd7c13c2c4cc35ac5fb8f#diff-46cc2ea1abbb0a6dfeeb5ec6efd89be6